test/index.js

```diff
@@ -475,6 +550,21 @@ describe('appengine', function() {
         });
       });
 
+      it('uses getRemoteApiRequestOptions_ to get the request options and passes them to sendHttpRequest_', function(done) {
+        var ae = new appengine.AppEngine();
+        var requestOptions = {};
```

Comment:
We're using strictEqual in the test, so only the identity of the object matters.